virtwho/virt/esx/esx.py

```diff
@@ -529,6 +529,14 @@ def __init__(self, *args, **kwargs):
 
     def _validate_server(self, key):
         error = super(EsxConfigSection, self)._validate_server(key)
+        if error is None:
+            try:
```

Comment:
Can you please add here the note mentioned in your comment on bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1727203#c1 It makes 100% sense, but it will not make any sense without this comment after one year ;-).